packages/govuk-frontend/src/govuk/components/character-count/character-count.mjs

```diff
@@ -88,7 +88,11 @@ export class CharacterCount extends GOVUKFrontendComponent {
         $textarea instanceof HTMLInputElement
       )
     ) {
-      return this
+      throw new ElementError($textarea, {
+        componentName: 'Character count',
+        identifier: '.govuk-js-character-count',
+        expectedType: 'HTMLTextareaElement" or "HTMLInputElement'
```

Comment:
**nitpick**: That's a bit meh to have these closing/opening quotes in the middle of the string (option 1). Equally, it felt a bit meh to expand the types of the `expectedType` to `Array<typeof HTMLElement>` as it starts to make a bit too many situations to handle (option 2). 

Two alternatives to these two "meh" options:
3. drop the quotes. Are they really helpful? we're not quoting the identifier, for example
4. limit the `expectedType` to two types: `string|string[]`, focusing the constructor on working with strings, either injecting as is or joining with `,` and a final `or`.

1 works but is a bit of a sore sight, 2 looks like more complexity than we need for the formatting of an error message, 4 looks a bit less complex (but hit by proper handling of 1, 2, and more than 2 cases) so 3 looks like the more straightforward path.

@colinrotherham @domoscargin how does that sound? @domoscargin would updating anything on `expectedType` affect any work on the Tabs?

Reply:
No impact on Tabs as far as I've got.

Think (3) sounds OK to me.

Reply:
Happy to drop the quotes, we don't use them for selectors

There are sneaky `string-but-maybe-already-array-to-array` things you can do to allow arrays too:

```
[expectedType].flat().join()
```

Reply:
Cool, I'll drop the quotes then, that'll make things simpler for now :) 

I think we're not quite geared for using `.flat()` yet as [we'd need a polyfill](https://browsersl.ist/#q=supports+es6-module+and+not+supports+array-flat%0A). We could work around it with a `Array.isArray(expectedType) ? expectedType.join(', ) : expectedType` (also more explicit, I think). Open for discussion in #4072.